data/operators/amenity/police.json

```diff
@@ -524,8 +513,7 @@
       "tags": {
         "amenity": "police",
         "operator": "Guardia di Finanza",
-        "operator:wikidata": "Q1552861",
-        "operator:wikipedia": "it:Guardia di Finanza"
```

Comment:
It is not redundant

you may think a single point not have a wikipedia pages? but infact it means a chain/series brand/amenity. so have this tag can help people know this amenity more easy and adequate

Reply:
That was only a question we discussed in the list for the StreetComplete quest, but I can add all the Wikipedia tags, I have nothing against it. Indeed I'll just add them all again (I will also add the ones which were not there).

Reply:
Thanks @naposm it would probably be better if you add them back.  Like @LaoshuBaby said, our build_wikidata script will try to add them back anyway, and it might pick a non-Italian sitelink.